src/v3/src/components/PhoneAuthenticator/PhoneAuthenticator.test.tsx

```diff
@@ -73,7 +70,7 @@ describe('PhoneAuthenticator tests', () => {
     props = getComponentProps({ targetKey: 'methodType' });
     const { findByLabelText, user } = setup(<PhoneAuthenticatorControl {...props} />);
 
-    await findByLabelText(/Country/);
+    await findByLabelText(/country.label/);
```

Comment:
this test does not looks correct, is the i18n key rendered on form, or due to incorrect mock?